### ISSUE TITLE
Integrate Broid to focus on higher level features

### ISSUE THREAD

**Issue by killix** [Bottr-js/Bottr#79]
After discussed with @jcampbell05. It will be a good idea to use @broidHQ [librairies](https://github.com/broidHQ/integrations) to integrate easily messaging platforms.

Actually Broid support:  Slack, Twitter, Viber, Line, Callr, Twilio, Skype, Discord, Messenger, Telegram, Kik and Flowdock. Coming this month: IRC, Email, Alexa

This work can fix quickly request feature like https://github.com/Bottr-js/Bottr/issues/73

I create the issue to track the work in progress and the futures PRs associated.

**Comment by jcampbell05** [comment]
@killix do you have support for web sockets at all ?

**Comment by killix** [comment]
you want to use web sockets to communicate with Broid libraries?

**Comment by jcampbell05** [comment]
@killix yeah so we currently have a custom web UI which uses web sockets to communicate, I guess I could build a client for that.
